Curve-finance-suite/Pool-Migration/src/agent.spec.ts

```diff
@@ -0,0 +1,199 @@
+import {
```

Comment:
It could be useful to add a test case calling the correct function but in a different contract.

Reply:
Done.